### ISSUE TITLE
.info/serverTimeOffset does not work in latest firebase-admin@8.4.0

### ISSUE THREAD

**Issue by rorymc** [firebase/firebase-admin-node#638]
Ian from Firebase Support has also verified the issue started with v8.2.0 of the Node.js Admin SDK

problem:
* https://github.com/firebase/firebase-js-sdk/issues/2043
* doesn't seem to have a corresponding issue listed in `firebase-admin`, but this definitely is an issue in the latest firebase-admin@8.4.0 -- hence filing this issue, per Firebase Support suggestion.

workaround is:
* downgrade to firebase-admin@8.0.0 + firebase/database@0.3.20 using yarn dependencies

env:
  * Operating System version: runtime is in firebase functions
  * Firebase SDK version: whatever comes with firebase-admin@8.4.0
  * Firebase Product: firebase-admin
  * Node.js version: 8.15.0
  * NPM version: 6.9.0

Steps to reproduce:
* try to use `.info/serverTimeOffset`, and it fails as described in https://github.com/firebase/firebase-js-sdk/issues/2043

seems that `firebase-admin` is just a meta-package, and something went wrong around 8.2.0

**Comment by hiranya911** [comment]
In which version of `@firebase/database` are you seeing this bug? It seems the issue has been fixed on v0.5.0.

**Comment by rorymc** [comment]
@hiranya911 , thank you for the question -- perhaps it's a broken issue with `firebase-admin` itself (being a meta-package of sorts), as it's not pulling in the proper `@firebase/database` higher than 0.3.11 for us.

we have no specific version of `@firebase-admin` pulled in -- as an example, this is the extent of the `firebase` related deps in our package.json:
```
    "firebase-admin": "8.4.0",
    "firebase-functions": "2",
```

**Comment by hiranya911** [comment]
I think it's because the database package has a 0.x.x version. In any case this will get resolved once #635 is released.

**Comment by rorymc** [comment]
thank you so much for triaging and helping us figure out what is going on.  helps with us creating a workaround, in the meantime.
